meta/main.yml

```diff
@@ -3,8 +3,14 @@ galaxy_info:
   author: Shane Frasier
   description: Install server setup script.
   company: CISA Cyber Assessments
+  galaxy_tags: []
```

Comment:
Tags?  We don't need no stinkin' tags!  Or maybe we do.

Reply:
There were no tags before.  Happy to add tags if you can think of any that apply.  I couldn't think of any since this role installs a specific tool of limited acclaim.  I didn't think that `serversetup` would mean anything to anyone.

Reply:
That's fine- I was just thinking `server` and `setup`, but that is fairly vague, so I'm ok with no tags if you are.